tsl/src/continuous_aggs/create.c

```diff
@@ -743,6 +745,7 @@ caggtimebucket_validate(CAggTimebucketInfo *tbinfo, List *groupClause, List *tar
 	{
 		SortGroupClause *sgc = (SortGroupClause *) lfirst(l);
 		TargetEntry *tle = get_sortgroupclause_tle(sgc, targetList);
+
```

Comment:
extra line

Reply:
I know we should try to avoid reformatting code in our normal PRs but this is a minor change and just to make the code a bit more readable. Also added an extra line in `cagg_validate_query` .